src/main/java/com/minecolonies/colony/buildings/BuildingFisherman.java

```diff
@@ -6,73 +6,98 @@
 import com.minecolonies.colony.ColonyView;
 import com.minecolonies.colony.jobs.AbstractJob;
 import com.minecolonies.colony.jobs.JobFisherman;
-import io.netty.buffer.ByteBuf;
-import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.util.BlockPos;
 
+/**
+ * The fishermans building.
+ */
 public class BuildingFisherman extends AbstractBuildingWorker
 {
+    /**
+     * The maximum upgrade of the building.
+     */
+    private static final int MAX_BUILDING_LEVEL = 3;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)